observation_portal/settings.py

```diff
@@ -199,8 +199,13 @@ def get_dramqtiq_broker_url():
 CORS_ORIGIN_ALLOW_ALL = True
 CORS_ALLOW_CREDENTIALS = True
 CORS_URLS_REGEX = r'^/(api|accounts|media)/.*$|^/o/.*'
-CORS_ORIGIN_WHITELIST = get_list_from_env('CORS_ORIGIN_WHITELIST')
-CSRF_TRUSTED_ORIGINS = get_list_from_env('CSRF_TRUSTED_ORIGINS')
+CORS_ALLOW_CREDENTIALS = True
```

Comment:
This line already exists a few lines up

Reply:
Sorry about all these changes to `settings.py` That was unintentional. I've reverted them.